src/components/ivaBrowser/ivaBrowser.tsx

```diff
@@ -0,0 +1,150 @@
+import { Alert, Col, Container, Row, Spinner } from "react-bootstrap";
+import { useAuth } from "../../services/auth";
+import { EmbeddedIVA } from "../../models/ivas";
+import { useEffect, useState } from "react";
+import { useMessages } from "../messages/usage";
+import {
+  FILTER_MAX_ISO,
+  FILTER_MIN_ISO,
+  WPS_URL,
+  fetchJson,
+} from "../../utils/utils";
+import IVABrowserList from "./ivaBrowserList/ivaBrowserList";
+import IVABrowserFilter from "./ivaBrowserFilter/ivaBrowserFilter";
+
+const IVABrowser = () => {
+  const [ivas, setIVAs] = useState<EmbeddedIVA[] | null | undefined>(undefined);
+
+  const { showMessage } = useMessages();
+  const { user } = useAuth();
+
+  let filteredIVAs: EmbeddedIVA[] | undefined = undefined;
+
+  const [filterObj, setFilterObj] = useState({
+    userFilter: "",
+    fromFilter: FILTER_MIN_ISO,
+    untilFilter: FILTER_MAX_ISO,
+    statusFilter: "",
+  });
+
+  function handleFilter(
+    user?: string,
+    from?: string,
+    until?: string,
+    status?: string
+  ) {
+    if (ivas) {
+      let userString: string =
+        user !== undefined ? user : filterObj["userFilter"];
```

Comment:
Can you make the filtering case insensitive (I think it's done like that in the access request browser already)?